### PR TITLE
UI issues: #175 #403 #139

### PR DESCRIPTION
**Pull request by victorpavlenko**
https://github.com/webRunes/Plus-WRIO-App/issues/175
https://github.com/webRunes/WRIO-InternetOS/issues/403
https://github.com/webRunes/Default-WRIO-Theme/issues/139

### CODE REVIEW

**Comment by AlexeyAnshakov** [comment]
@michbil please review

File: WRIO-InternetOS/js/components/CreateDomCenter.js [hunk 1]
```diff
@@ -1,3 +1,10 @@
+'use strict';
```
Comment (michbil) [inline WRIO-InternetOS/js/components/CreateDomCenter.js]: @Xurma2 No need to 'use strict', it's assumed by default in ES6

File: WRIO-InternetOS/js/components/CreateDomCenter.js [hunk 2]
```diff
@@ -57,8 +67,6 @@ class CreateDomCenter extends React.Component{
         });
         this.listenStoreMenuSidebar = StoreMenu.listenTo(ActionMenu.showSidebar, this.onShowSidebar);
 
-        var that = this;
-
         CenterActions.gotWrioID.listen( function(id) {
```
Comment (michbil) [inline WRIO-InternetOS/js/components/CreateDomCenter.js:70]: @Xurma2 You have deleted `that = this` assignment, but using that in `that.getAuthorWrioID());`
There gonna be error there

Comment (victorpavlenko) [inline WRIO-InternetOS/js/components/CreateDomCenter.js:70]: See 54 line

File: WRIO-InternetOS/js/components/CreateDomCenter.js (hunk 1 above)
Comment (victorpavlenko) [inline WRIO-InternetOS/js/components/CreateDomCenter.js]: Removed

File: WRIO-InternetOS/js/components/CreateDomCenter.js (hunk 2 above)
Comment (michbil) [inline WRIO-InternetOS/js/components/CreateDomCenter.js:70]: @Xurma2 ok, see now, code was collapsed.

File: WRIO-InternetOS/js/components/CreateDomCenter.js (hunk 1 above)
Comment (michbil) [inline WRIO-InternetOS/js/components/CreateDomCenter.js]: @Xurma2 why then i still see this line?

Comment (victorpavlenko) [inline WRIO-InternetOS/js/components/CreateDomCenter.js]: I do not commit later

2015-12-08 15:51 GMT+03:00 Michael Bilenko notifications@github.com:

Comment (victorpavlenko) [inline WRIO-InternetOS/js/components/CreateDomCenter.js]: I do not commit later

**Pull request merged**